AY2021S1-CS2103T-F11-4/tp
# Update documentation

## Description

According to team 2 reviews. 

Fixes #234 
Fixes #233 
Fixes #246 

## Testing

- Manual clicking on links.
- Image positionings
- Check help window
- Check links to command summary and parameter constraints summary

## Remarks

Add any additional remarks for others to take note of:

- UG references: 
   - [senior's work](https://ay1920s2-cs2103t-f10-1.github.io/main/UserGuide.html), take note of the mockup annotations, use of symbols and verboseness
